## On Solaris, libxslt passes some invalid flags to the
## linker, even though it correctly detects the linker
## as being Solaris instead of GNU. This removes that
## logic and leaves the version script flags unset

--- a/configure.ac  2019-04-24 10:31:11.000000000 +0300
+++  b/configure.ac  2019-04-24 10:31:56.000000000 +0300
@@ -133,11 +133,11 @@
 dnl
 AC_MSG_CHECKING([how to pass version script to the linker ($LD)])
 VERSION_SCRIPT_FLAGS=none
-if $LD --help 2>&1 | grep "version-script" >/dev/null 2>/dev/null; then
-    VERSION_SCRIPT_FLAGS=-Wl,--version-script=
-elif $LD --help 2>&1 | grep "M mapfile" >/dev/null 2>/dev/null; then
-    VERSION_SCRIPT_FLAGS="-Wl,-M -Wl,"
-fi
+#if $LD --help 2>&1 | grep "version-script" >/dev/null 2>/dev/null; then
+#    VERSION_SCRIPT_FLAGS=-Wl,--version-script=
+#elif $LD --help 2>&1 | grep "M mapfile" >/dev/null 2>/dev/null; then
+#    VERSION_SCRIPT_FLAGS="-Wl,-M -Wl,"
+#fi
 AC_MSG_RESULT([$VERSION_SCRIPT_FLAGS])
 AC_SUBST(VERSION_SCRIPT_FLAGS)
 AM_CONDITIONAL([USE_VERSION_SCRIPT], [test "$VERSION_SCRIPT_FLAGS" != none])

